frontend/src/pages/Users/UsersTable.js

```diff
@@ -88,8 +123,8 @@ const UsersTable = ({
         )}
       </Table>
     </Paper>
-  ) : (
+  ) : areUsersEnabled ? (
```

Comment:
It's difficult to read. Only use the conditional operator for one condition.

Use emptyState for disabled user list

Reply:
done